taxi_service/urls.py

```diff
@@ -21,5 +21,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("", include("taxi.urls", namespace="taxi")),
+    path("taxi/", include("taxi.urls", namespace="taxi")),
```

Comment:
why did you decide to change it?

Reply:
I figured that if there is more than one application, then the paths should match the names of the applications. And the empty one should lead to the home page.

Reply:
but the home page is in `taxi.urls` now so it works as you said.  And if we add new applications, we will do so, but so far there is only one